deps/openlibm.version

```diff
@@ -1,2 +1,2 @@
-OPENLIBM_BRANCH=v0.4.1
-OPENLIBM_SHA1=3ee2a6e6a9fc68cbf2ab00f830b9059215e40b06
+OPENLIBM_BRANCH=master
```

Comment:
So what is the meaning of `OPENLIBM_BRANCH`?  Would `OPENLIBM_TAG` be more appropriate?